engage/app/templates/calendar.hbs

```diff
@@ -1,6 +1,79 @@
 <div class="row">
   <div class="col-md-8 main-left">
-    Calendar
+    <h2 class="page-header">Calendar</h2>
+    <div class="media list-group-item calendar-list-item">
+      <div class="media-left">
+        <a href="#">
+          <img class="media-object" src="http://lorempixel.com/100/100/nature" alt="#">
+        </a>
+      </div>
+      <div class="media-body">
+        <h4 class="media-heading">Sustainability Series</h4>
+        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Laudantium ex maxime iure perspiciatis itaque aliquid ipsum.
+        </p>
+      </div>
+    </div>
+    <div class="media list-group-item calendar-list-item">
```

Comment:
`calendar-list-item` is pretty specific.... let's try to be more modular. we already have code that does what you're trying to do with that, i think